tfmigrate/config.go

```diff
@@ -14,10 +14,10 @@ type MigrationConfig struct {
 // MigratorConfig is an interface of factory method for Migrator.
 type MigratorConfig interface {
 	// NewMigrator returns a new instance of Migrator.
-	NewMigrator(o *MigratorOption) (Migrator, error)
+	NewMigrator(o *MigratorOption, isBackendTerraformCloud bool) (Migrator, error)
```

Comment:
Can we move the `isBackendTerraformCloud` flag into `MigratorOption`?
I hesitate to change the interface every time such a customization flag is added.

Reply:
Yep, that totally makes sense, will do.

Reply:
Update will involve updating the `MigratorOption` struct, but that definitely is still cleaner than changing the interface.